src/cards.csv

```diff
@@ -2,53 +2,53 @@
 "508","Incentives.","How are interviewees incentivised?","g","2","","",""
 "507","NDA interviewee","Does the interviewee need to sign an NDA?","g","2","","",""
 "506","Interview location","Where do the user interviews take place?","g","2","","",""
-"505","End user recruitment.","Who does the end user recruitment for user interviews?","g","3","","",""
-"504","Contact point client?","Who in the team is the contact point for the client?","g","3","","",""
-"503","Visuals in SVS?","Are visuals part of the SVS deliverables?","g","1","","",""
-"502","Prototypes in SVS?","Are prototypes part of the SVS deliverables?","g","1","","",""
-"501","Right people.","Can we have the right people to cover all aspects in the SVS: desirability, feasibility, viability?","g","3","","",""
-"500","Client in SVS.","Who of the client is part of the SVS and how much?","g","3","","",""
-"499","Client in SVS","Is the client part of the SVS?","g","3","","",""
+"505","End user recruitment.","Who does the end user recruitment for user interviews?","g","3","Who recruits the users?","",""
+"504","Contact point client?","Who in the team is the contact point for the client?","g","3","Who is the key client contact?","",""
+"503","Visuals in project?","Are visuals part of the deliverables?","g","1","","",""
+"502","Prototypes in project?","Are prototypes part of the deliverables?","g","1","","",""
+"501","Right people.","Do we have the right people to cover all aspects: desirability, feasibility, viability?","g","3","","",""
+"500","Client in project.","How much time will each stakeholder have for the project?","g","3","","",""
```

Comment:
Let's change the title to Client availability

Reply:
👍